src/validation/tests/recursive_validation_tests.rs

```diff
@@ -699,3 +702,23 @@ mod mixed_structs_and_functionblocks {
         assert_snapshot!(&diagnostics);
     }
 }
+
+mod enums {
+    use crate::test_utils::tests::parse_and_validate_buffered;
+
+    #[test]
+    fn enums_are_not_considered_for_duplicate_checks() {
+        //... because they're integers (duh)
+        let diagnostics = parse_and_validate_buffered(
+            "
+            FUNCTION_BLOCK FOO
+            VAR
+                foo_enum : (start, stop) := stop;
```

Comment:
I think you forgot to add the duplicate to the test here.

e.g.
```
foo_enum : (start, stop) := stop;
bar_enum : (stop, go) := stop;
```

Reply:
I see, because both variants have the same datatype which would then be added to the path for each variant if not differentiating between struct members and enum members. Guess at first glance it just looked weird to speak of duplicates when only a single enum is present in the test.